spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe "training without cards" do
+  before :all do
+    Card.destroy_all
+    Card.create(original_text: 'дом', translated_text: 'house')
+    @card = Card.pending.first
+  end
+
+  it 'no cards' do
+    visit '/'
+    expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+  end
+end
+
+describe "training with some cards" do
+  before :each do
+    Card.destroy_all
+    @card = Card.create(original_text: 'дом', translated_text: 'house')
+    @card.update_attribute(:review_date, Time.now)
+    @card = Card.create(original_text: 'машина', translated_text: 'car')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.